core/src/main/res/values/strings.xml

```diff
@@ -41,6 +41,21 @@
     <string name="core_date_format_MMMM_dd">MMMM dd</string>
     <string name="core_full_date_with_time">dd MMM yyyy hh:mm aaa</string>
 
+    <!-- region: Course Date types -->
+    <!-- Label used when the date is Today -->
+    <string name="date_type_today" translatable="false">Today</string>
```

Comment:
Why are they not translatable? 🤔

Reply:
added by mistake, replaced with `tools:ignore="MissingTranslation"` cuz don't have translation for now.